src/main/java/cn/nukkit/permission/PermissibleBase.java

```diff
@@ -38,7 +39,7 @@ public boolean isOp() {
     }
 
     @Override
-    public void setOp(boolean value) {
+    public void setOp(@NotNull boolean value) {
```

Comment:
This `@NotNull` annotation is not needed because `boolean` is a primitive type and will never be `null`. The others are fine.